### ISSUE TITLE
Internet Explorer JS failure

### ISSUE THREAD

**Issue by merrill-oakland** [wiris/moodle-atto_wiris#49]
While IE is ancient, we have a med school that has to use it in some locations, and it's still supported by Moodle.

If the Wiris atto plugin in installed, then IE throws a fatal JS error and stops the rest of the page from loading. The easiest place we have to see this is to go to add a File resource to a course. The file picker will never load.

The exception given is `SCRIPT5007: Object.keys: argument is not an Object`

And seems to point to this line in the combined/minified JS:
```
{key:"httpBuildQuery",value:function(t){var i="";return Object.keys(t).forEach(function(n){null!=t[n]&&(i+="".concat(e.urlEncode(n),"=").concat(e.urlEncode(t[n]),"&"))}),"&"===i.substring(i.length-1)&&(i=i.substring(0,i.length-1)),i}
```

**Comment by manuelwiris** [comment]
Hi @merrill-oakland, and sorry for the delayed answer.

The issue has been added to our backlog. We will keep you udpated.

Thanks for your feedback

**Comment by merrill-oakland** [comment]
I was contacted by Wiris support and told that this was fixed, likely in 7.16.0.

I can confirm that it is fixed for us - sometime between 7.12.0 and 7.16.0.

Closing.
